pyof/v0x04/symmetric/experimenter.py

```diff
@@ -49,4 +49,4 @@ def __init__(self, xid=None, experimenter=None, exp_type=None, data=b''):
         super().__init__(xid)
         self.experimenter = experimenter
         self.exp_type = exp_type
-        self.data = data
+        self.data = BinaryData() if data is None else data
```

Comment:
`BinaryData()` -> `b''`; it's bad having GenericTypes when they are not needed.
The cleaner way is to set `b''` directly in the `__init__` method - same behavior, better readability.